hasher-matcher-actioner/webapp/src/pages/SubmitContent.jsx

```diff
@@ -0,0 +1,346 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ */
+
+import React, {useState} from 'react';
+import {
+  Col,
+  Button,
+  Row,
+  Form,
+  Container,
+  InputGroup,
+  Image,
+  Collapse,
+  Accordion,
+  Card,
+  Spinner,
+} from 'react-bootstrap';
+import {submitContent, submitContentUpload} from '../Api';
+
+const SUBMISSION_TYPE = Object.freeze({
+  UPLOAD: 'Direct Upload',
+  RAW: 'Raw Value (example only)',
+  S3_OBJECT: 'S3 Object (example only)',
+  PRESIGNED_URL: 'Presigned URL (example only)',
```

Comment:
I think just URL might be an option, too. If a partner is using a VPC, they might not bother with needing to go through the trouble of pre-signing a URL. Also, signed URLs may expire. It might work best for a partner for the URL to be available longer / indefinitely... Anyways, I suggest adding URL as another submission type.

Reply:
thanks for the feedback, this PR was still in draft mode when you reviewed so the history inline won't be tracked because I am force pushing.

Reply:
changed PRESIGNED_URL to just URL for now (doesn't matter because neither work yet haha)